_posts/2019-07-08-persistent-storage-improvements.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title:  "Persistent storage improvements"
+date: 2019-08-07
+author: jakub_scholz
+---
+
+Few months ago I wrote [a blog post](https://strimzi.io/2019/02/28/resizing-persistent-volumes.html) about how you can manually increase the size of your persistent volumes used for Kafka or Zookeeper storage.
+I promised there that one day it will be supported directly in Strimzi.
+And exactly that happened in the [Strimzi 0.12 release](https://github.com/strimzi/strimzi-kafka-operator/releases).
```

Comment:
The link is to the releases page not to the specific 0.12.1 release.

Reply:
This was really intentional so that we lead the users to the latest releases rather then one specific. I would probably prefer to keep the link generic.